sys/fs/fuse/fuse_vnops.c

```diff
@@ -2624,10 +2626,10 @@ fuse_vnop_getextattr(struct vop_getextattr_args *ap)
 	int err;
 
 	if (fuse_isdeadfs(vp))
-		return (ENXIO);
+		return (EXTERROR(ENXIO, "This FUSE session is about to be closed"));
 
 	if (fsess_not_impl(mp, FUSE_GETXATTR))
-		return EOPNOTSUPP;
+		return (EXTERROR(EOPNOTSUPP, "This daemon does not implement extattrs"));
```

Comment:
Use "server" for consistency.  And spell out "extended attributes".
```suggestion
		return (EXTERROR(EOPNOTSUPP, "This server does not implement extended attributes"));
```